flocker/control/test/test_model.py

```diff
@@ -721,6 +721,18 @@ class Record(PRecord):
                  Record().value2.__class__.__name__) ==
                 ("SomethingAnotherPMap", "IntFloatPMap"))
 
+    def test_additional_keyword_arguments(self):
+        """
+        The ``invariant`` parameter is passed through to ``field``.
+        """
+        class Record(PRecord):
+            value = pmap_field(
+                int, int,
+                invariant=lambda pmap: (False, "Very strict invariant.")
+            )
+
+        self.assertRaises(InvariantException, Record, value={})
```

Comment:
This doesn't seem quite enough to prove it's using the invariant - maybe use an invariant the passes some things but not others, so you can check both positive and negative invariant use paths?